enviroment.rb

```diff
@@ -0,0 +1,80 @@
+require 'securerandom'
+
+module Enviroment
+  class Depot
+    attr_accessor :packs
+
+    def initialize
+      @packs = {
+        simple_transportation_pack: { intelligence: [:cellphone], items: %i[medipack chevy_versa] },
```

Comment:
Great! It is very good you use as few lines as possible without risking the code may not be unreadable.